content/en/docs/configuration/acme/http01/_index.md

```diff
@@ -155,3 +168,167 @@ The added labels and annotations will merge on top of the cert-manager defaults,
 overriding entries with the same key.
 
 No other fields of the ingress can be edited.
+
+## Configuring the HTTP-01 Gateway API solver
+
+**FEATURE STATE**: cert-manager 1.5 [alpha]
+
+The Gateway and HTTPRoute resources are part of the [Gateway API][gwapi], a set
+of CRDs that you install on your Kubernetes cluster that provide various
+improvements over the Ingress API.
+
+[gwapi]: https://gateway-api.sigs.k8s.io
+
+{{% pageinfo color="info" %}}
+
+📌  This feature requires the installation of the Gateway API CRDs and passing a
+feature flag to the cert-manager controller.
+
+To install the Gateway API CRDs, run the following command:
+
+```sh
+kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.3.0" | kubectl apply -f -
+```
+
+To enable the feature in cert-manager, turn on the `GatewayAPI` feature gate:
+
+- If you are using Helm:
+
+  ```sh
+  helm upgrade --install cert-manager jetstack/cert-manager --namespace cert-manager \
+    --set "extraArgs={--feature-gates=ExperimentalGatewayAPISupport=true}"
+  ```
```

Comment:
Random thought: If you'd previously enabled the experimental CertificateRequestSupport would this command disable that. I do not propose changing this command, but it is worth thinking about how easy it is to change `extraArgs` values during an upgrade.

Reply:
Good point, I wish we could instead re-use the same flag multiple times:

```sh
  helm upgrade --install cert-manager jetstack/cert-manager --namespace cert-manager \
    --set extraArgs="--feature-gates=ExperimentalGatewayAPISupport=true"
    --set extraArgs="--feature-gates=ExperimentalCertificateRequestSupport=true"
```

But the only way to set arrays from the command line is to use the syntax `extraArgs={...,...}` 😞